test/lint/extended-lint-cppcheck.sh

```diff
@@ -55,7 +55,6 @@ IGNORED_WARNINGS=(
     "src/test/checkqueue_tests.cpp:.* Struct 'UniqueCheck' has a constructor with 1 argument that is not explicit."
     "src/test/fuzz/util.h:.* Class 'FuzzedFileProvider' has a constructor with 1 argument that is not explicit."
     "src/test/fuzz/util.h:.* Class 'FuzzedAutoFileProvider' has a constructor with 1 argument that is not explicit."
-    "src/util/ref.h:.* Class 'Ref' has a constructor with 1 argument that is not explicit."
```

Comment:
2e3b6404235cf753bd130fb58a4cb0022c06b8a0 : duplicate https://github.com/dashpay/dash/pull/5974/commits/5d8abea6bdb5734641836c207f072e6a171b7b73 in #5974

Reply:
removed from 5974